flutter/pubspec.yaml

```diff
@@ -92,6 +92,7 @@ dependencies:
     password_strength: ^0.2.0
     flutter_launcher_icons: ^0.11.0
     flutter_keyboard_visibility: ^5.4.0
+    texture_rgba_renderer: ^0.0.8
```

Comment:
The reason why the plugin is needed, is that the `rustdesk_desktop_multi_window` has injected this plugin for desktop platforms, we need to pass the compilation.